lib/Runtime/Library/JavascriptNumber.cpp

```diff
@@ -585,19 +585,23 @@ namespace Js
         {
             //use the first arg as the fraction digits, ignore the rest.
             Var aFractionDigits = args[1];
+            bool noRangeCheck = false;
 
             // shortcut for tagged int's
             if(TaggedInt::Is(aFractionDigits))
             {
                 fractionDigits = TaggedInt::ToInt32(aFractionDigits);
             }
             else if(JavascriptOperators::GetTypeId(aFractionDigits) == TypeIds_Undefined)
-                ; // fraction digits = -1
+            {
```

Comment:
That will make the code behave a little different (like before the refactor). `fractionDigits` as `TaggedInt` will not be checked.

---
In reply to: [100205438](https://github.com/Microsoft/ChakraCore/pull/2497#discussion_r100205438) [](ancestors = 100205438)